torch/distributed/checkpoint/default_planner.py

```diff
@@ -234,6 +234,29 @@ def __init__(self, keys=None, *args, **kwargs):
         self.keys = keys
         super().__init__(*args, **kwargs)
 
+    def _should_include_key(self, key: str, metadata: Metadata) -> bool:
+        if self.keys is None:
+            return True
+
+        if key in self.keys:
+            True
+
+        unflattened_keys = []
```

Comment:
Nit: I wonder if something like "unflattened_intermediary_paths" could be a more intuitive name for this variable?